integration/test/010_projects.Tests.ps1

```diff
@@ -186,6 +186,64 @@ Describe 'VSTeam Integration Tests' -Tag 'integration' {
       }
    }
 
+   Context 'ReleaseDefinition full exercise' {
+      BeforeAll {
+
+         if ($acct -like "http://*") {
+            $defaultQueue = Get-VSTeamQueue -ProjectName $newProjectName | Where-Object { $_.poolName -eq "Default" }
+         }
+         else {
+            $defaultQueue = Get-VSTeamQueue -ProjectName $newProjectName | Where-Object { $_.poolName -eq "Hosted" }
+         }
+
+         $srcReleaseDef = Get-Content "$PSScriptRoot\sampleFiles\010_releasedef_1.json" -Raw | ConvertFrom-Json
+         $srcReleaseDef.name = $newProjectName + "-CD1"
+         $srcReleaseDef.environments[0].deployPhases[0].deploymentInput.queueId = $defaultQueue.Id
+         $tmpReleaseDef1 = (New-TemporaryFile).FullName
+         $srcReleaseDef | ConvertTo-Json -Depth 10 | Set-Content -Path $tmpReleaseDef1
+
+         Invoke-VSTeamRequest -ProjectName $newProjectName -Method Post -SubDomain vsrm -Area Release -Resource definitions -Version "5.1" -InFile $tmpReleaseDef1
```

Comment:
Hard coding the  version will cause these tests to fail on TFS.

Reply:
I am a bit confused of what you are trying to do here. Why are you using Invoke-VSTeamRequest?

Reply:
When the version is corrected it appears to work on TFS2018 and above.

Reply:
I think have the tests fixed now.

Reply:
Thanks for correcting! I don't know anymore why I used the wrong command for testing release definitions. Doesn't make sense to me anymore. Maybe a brain hickup...